playwright.config.ts

```diff
@@ -1,4 +1,9 @@
 import type { PlaywrightTestConfig } from '@playwright/test';
+import dotenv from 'dotenv';
+
+if (process.env.NODE_ENV === 'development') {
+  dotenv.config({ path: '.env' });
+}
 
 const config: PlaywrightTestConfig = {
```

Comment:
Let's configure an expect timeout of 10 seconds instead of 5 seconds (we've had some flaky tests for that in the recent past)